### PR TITLE
Kubernetes add nodename and check fuse.conf

### PR DESCRIPTION
**Pull request by johscheuer**
This PR adds the following features:

- Check if fuse.conf exists and contains `user_allow_other`
- Give Services the name of the Node name

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -21,6 +23,34 @@ EOF
...
+
+# Set the maximum number of FUSE mounts allowed to non-root users.
+# The default is 1000.
+#mount_max = 1000
+
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
+```
+
+Felix Hupfeld (felix@quobyte.com) bitte dazu! mach doch ein grep und echo (roberts idee) es muss user_allow_other rein
```
Comment (quobert) [inline kubernetes/README.md]: remove

File: kubernetes/README.md [hunk 2]
```diff
@@ -21,6 +23,32 @@ EOF
...
+
+```bash
+$ cat ./fuse.conf
+# Copied from Quobyte Client Container
+
+# /etc/fuse.conf - Configuration file for Filesystem in Userspace (FUSE)
+
+# Set the maximum number of FUSE mounts allowed to non-root users.
+# The default is 1000.
+#mount_max = 1000
```
Comment (quolix) [inline kubernetes/README.md]: Remove this section.

**Comment by johscheuer** [comment]
Updated and rebased

File: kubernetes/client-ds.yaml [hunk 3]
```diff
@@ -18,16 +18,22 @@ spec:
           - -xec
           - |
             ADDR=$(echo $(nslookup ${QUOBYTE_REGISTRY} | grep -A10 -m1 -e 'Name:' | grep Address | awk '{split($0,a,":"); print a[2]}'  | awk '{print $1":7866"}') | tr ' ' ,)
+            if [[ ! -f /rootfs/etc/fuse.conf ]]; then
+              echo "Copy fuse config to host"
+              { echo -e '# Copied from Quobyte Client Container\n'; cat /etc/fuse.conf; } > /rootfs/etc/fuse.conf
+            fi
+            if [[ $(grep "^[^#]" /etc/fuse.conf | grep -c "user_allow_other" /etc/fuse.conf) -eq 0 ]]; then
```
Comment (quobert) [inline kubernetes/client-ds.yaml]: check /rootfs/etc/fuse.conf instead of /etc/fuse.conf

Comment (johscheuer) [inline kubernetes/client-ds.yaml]: missed this one

**Comment by quolix** [comment]
LGTM from us

**Pull request merged**